happi/backends/qs_db.py

```diff
@@ -0,0 +1,151 @@
+"""
+Backend implementation for parsing the Questionnaire
+"""
+import re
+import logging
+
+from psdm_qs_cli import QuestionnaireClient
+
+from .json_db import JSONBackend
+from ..errors import DatabaseError
+
+logger = logging.getLogger(__name__)
+
+
+class QSBackend(JSONBackend):
```

Comment:
I'll approve without any changes here, but it might be better organizationally to have `QSBackend` and `JSONBackend` both inherit from a shared parent rather than from each other. The code will be easier to maintain if the `QSBackend` class defines only how to get data from the Questionnaire and the `JSONBackend` defines only how to get data from a json file, with some other class defining the shared features that these and similar backends would need.

Reply:
I agree. I think that's the correct way. I'll mark it as an issue for later review.